packages/flutter/lib/src/material/refresh_indicator.dart

```diff
@@ -143,6 +143,7 @@ class RefreshIndicator extends StatefulWidget {
     this.semanticsValue,
     this.strokeWidth = RefreshProgressIndicator.defaultStrokeWidth,
     this.triggerMode = RefreshIndicatorTriggerMode.onEdge,
+    this.elevation = 2,
```

Comment:
Really really tiny nit: Since the parameter is a double, it should have a default of 2.0 instead of 2.

Reply:
@MitchellGoodwin - Not a problem. This is tiny but a very valid observation. Thanks for noticing. Pushed a new commit.